docs/content/operator/gas-station/architecture/features.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Gas Station Features
+description: Learn about the key features of the IOTA Gas Station
+teams:
+  - iotaledger/identity
+tags:
+  - gas-station
+  - sponsored-transactions
+  - explanation
+  - reference
+---
+
+## Access Controller
+
+The **Access Controller** is a rule-based system that regulates access to the `/execute_tx` endpoint. Similar to firewalls, it enforces policies to determine which transactions are allowed or denied.
+
+### Access Policy Modes
+
+The Access Controller operates in one of two modes:
+
+* **Deny All:** A strict security model where all transactions are blocked by default. Rules must be explicitly defined to allow specific transactions.
+* **Allow All:** A more permissive model where all transactions are allowed by default. Rules are used to selectively deny certain transactions.
+
+:::warning Important Note
+While the **`/reserve_gas`** endpoint remains unprotected by the ACL, potential abuse (such as multiple reservation attacks) is mitigated through configuration options like maximum reservation limits and expiration times.
+:::
+
+### Rule Processing
+
+The Access Controller processes rules sequentially, much like rule-based filtering systems such as firewalls. Each rule is evaluated in order, and once a matching rule is found, its action is applied, preventing further rules from being checked. This ensures predictable behavior where the most specific rules are placed earlier, while broader rules serve as fallbacks.
+
+When a transaction is submitted, the Access Controller starts by checking the first rule in the list. If the transaction matches the conditions set in that rule, the corresponding action is taken, either allowing or denying the transaction. If no match is found, the system proceeds to the next rule in sequence. This process continues until either a rule applies or no rules match, in which case the system falls back to the default policy (either **Allow All** or **Deny All**).
+
+### Example
+
+For example, consider the following **Allow All** policy configuration with three rules:
+
+```yaml
+# Rule 1
+- sender-address: 0x0202020202020202
```

Comment:
Hm, could we use valid address examples, similar to the more recent update to the gas station Readme?

E.g. use `0x0202020202020202020202020202020202020202020202020202020202020202` instead of `0x0202020202020202` here.